gpiozero/boards.py

```diff
@@ -1185,3 +1185,86 @@ def on(self):
     def off(self):
         self.value = False
 
+class SevenSegmentDisplay(LEDBoard):
```

Comment:
> I was thinking also display the character represented

If that _was_ to be added, wouldn't that be a better fit for `__str__` rather than `__repr__` ? \*shrug\*